bundles/org.openhab.binding.dirigera/src/main/java/org/openhab/binding/dirigera/internal/handler/light/ColorLightHandler.java

```diff
@@ -0,0 +1,270 @@
+        super.handleCommand(channelUID, command);
+        String channel = channelUID.getIdWithoutGroup();
+        if (CHANNEL_LIGHT_COLOR.equals(channel)) {
+            if (command instanceof HSBType hsb) {
+                // respect sequence
+                switch (lightConfig.fadeSequence) {
+                    case 0:
+                        brightnessCommand(hsb);
+                        colorCommand(hsb);
+                        break;
+                    case 1:
+                        colorCommand(hsb);
+                        brightnessCommand(hsb);
+                        break;
+                }
+                hsbStateReflection = hsb;
+                updateState(channelUID, hsb);
+            } else if (command instanceof OnOffType) {
+                super.addOnOffCommand(OnOffType.ON.equals(command));
+            } else if (command instanceof PercentType percent) {
+                int requestedBrightness = percent.intValue();
+                if (requestedBrightness == 0) {
+                    super.addOnOffCommand(false);
+                } else {
+                    brightnessCommand(new HSBType("0,0," + requestedBrightness));
+                    super.addOnOffCommand(true);
+                }
+            }
+        }
+        if (CHANNEL_LIGHT_TEMPERATURE.equals(channel)) {
+            if (command instanceof PercentType percent) {
+                // there are color lights which cannot handle temperature as stored in capabilities
+                // in this case calculate color which is fitting to temperature
+                // otherwise TemperatureLightHandler will do the command
+                if (!receiveCapabilities.contains(Model.COLOR_TEMPERATURE_CAPABILITY)) {
+                    int kelvin = super.getKelvin(percent.intValue());
+                    HSBType colorTemp = getHSBTemperature(kelvin);
+                    HSBType colorTempAdaption = new HSBType(colorTemp.getHue(), colorTemp.getSaturation(),
+                            hsbDevice.getBrightness());
+                    if (customDebug) {
+                        logger.info("DIRIGERA COLOR_LIGHT {} handle temperature as color {}", thing.getLabel(),
+                                colorTempAdaption);
+                    }
+                    colorCommand(colorTempAdaption);
+                }
+            }
+        }
+    }
+
+    /**
+     * Send hue and saturation to light device in case of difference is more than 2%
+     *
+     * @param hsb as requested color
+     * @return true if color request is sent, false otherwise
+     */
+    private void colorCommand(HSBType hsb) {
+        // if (!hsb.closeTo(hsbDevice, 0.02)) {
+        /**
+         * Why is closeTo not working on target?
+         * 2024-11-09 09:28:21.191 [TRACE] [rnal.handler.light.ColorLightHandler] - DIRIGERA COLOR_LIGHT hsb too
+         * close Device 63.0,0,33 Requested 63,100,0
+         **/
```

Comment:
Use it or drop it, or start a discussion if you need guidance. (unfortunately i'm not expert in the specific color stuff).

Reply:
See this https://github.com/openhab/openhab-core/issues/4655#issuecomment-2748685090